starboard/CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ since the version previous to it.
 
 ## Version 16
 
+### Create POSIX socket API emu layer for Win32
```

Comment:
Please re-write as:

### Added standard POSIX socket/close APIs.

The standard API `socket` can be used  from `<sys/socket.h>` and
`close` can be called  to close the socket by including `<unistd.h>`.